hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/configuration/FlinkOptions.java

```diff
@@ -614,6 +614,13 @@ private FlinkOptions() {
       .defaultValue(ClientIds.INIT_CLIENT_ID)
       .withDescription("Unique identifier used to distinguish different writer pipelines for concurrent mode");
 
+  public static final ConfigOption<Boolean> WRITE_ALLOW_COMMIT_ON_EMPTY_BATCH = ConfigOptions
+      .key("write.allow.commit.on.empty.batch")
+      .booleanType()
```

Comment:
Can we remove this option and add a tool method for option `hoodie.allow.empty.commit` in `OptionsResolver` ?

Reply:
Sure! will update and push later